### PR TITLE
XS✔ ◾ Create autocorrect-in-outlook

### PR DESCRIPTION
**Pull request by ae-cher-ssw**
Creating a new rule per Uly email request RE: Microsoft Bookings form vs. current contact us form

### CODE REVIEW

File: rules/autocorrect-in-outlook [hunk 1]
```diff
@@ -0,0 +1,42 @@
+type: rule
+title: Do you know how to use Autocorrect in Outlook to insert common links you often send?
```
Comment (tiagov8) [inline rules/autocorrect-in-outlook:2]: Hi @ae-cher-ssw 

This title is too long.

- [ ] 1. What about changing to _"Do you use Autocorrect in Outlook to insert links you often send?"_

File: rules/autocorrect-in-outlook [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+uri: /rules/autocorrect-in-outlook
```
Comment (tiagov8) [inline rules/autocorrect-in-outlook:3]: Hi @ae-cher-ssw 

We don't have "/rules/" in the URI

- [ ] 1. Read https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Create-Rules
- [ ] 2. Please fix the URI

File: rules/autocorrect-in-outlook [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+authors:
+  - title: Ulysses Maclaren
+    url: https://www.ssw.com.au/people/uly
+created: 2024-01-08T13:42:55.753Z
+guid: 7a8b9c0d-3b2e-4f31-af5e-9f9f4a2d9d86
+---
+# Do you know how to use Autocorrect in Outlook to insert common links you often send?
```
Comment (tiagov8) [inline rules/autocorrect-in-outlook:10]: Hi @ae-cher-ssw 

You don't need to repeat the title in the content

- [ ] 1. Please remove this heading

File: rules/autocorrect-in-outlook [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+You just received an email from a client asking for your company's Terms and Conditions. Instead of navigating through your files or searching your website to find the link, wouldn't it be more efficient to simply type a shorthand code like `#T` and have Outlook automatically insert the URL for you? This is where the Autocorrect feature in Outlook can become a huge time-saver.
+
+<!--endintro-->
+
+## Why Use Autocorrect for URLs?
+
+- **Efficiency**: It saves time by eliminating the need to find and copy-paste URLs.
```
Comment (tiagov8) [inline rules/autocorrect-in-outlook:18]: Hi @ae-cher-ssw 

- [ ] 1. Please follow https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists/ in all your lists ;)

File: rules/autocorrect-in-outlook [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+::: greybox
+
+    #T -> your company’s Terms and conditions. E.g. https://www.ssw.com.au/terms-and-conditions 
+    #B -> your [Bookings with Me](https://www.ssw.com.au/rules/meeting-bookings) 
+
+:::
+
+::: good
+
+Figure: Good Example {Using shorthand codes to represent commonly used URLs}
```
Comment (tiagov8) [inline rules/autocorrect-in-outlook:38]: Hi @ae-cher-ssw 

Unsure why do you have curly brackets here...

- [ ] 1. Please change to _"Figure: Good example - Using shorthand codes to represent commonly used URLs"_

File: rules/autocorrect-in-outlook [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+<!--endintro-->
+
+## Why Use Autocorrect for URLs?
+
+- **Efficiency**: It saves time by eliminating the need to find and copy-paste URLs.
+- **Consistency**: It ensures that everyone is using the same URL and reduces the risk of errors.
+  
+## How to Set It Up
+
+1. **Open Outlook** and go to `File` -> `Options`.
```
Comment (tiagov8) [inline rules/autocorrect-in-outlook:23]: Hi @ae-cher-ssw 

- [ ] 1. Please follow https://ssw.com.au/rules/use-correct-symbols-when-documenting-instructions/

File: rules/autocorrect-in-outlook [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+2. Under the `Mail` tab, click on `Spelling and Autocorrect`.
+3. Choose `Autocorrect Options` and add your shorthand and URL.
+
+![Figure: Make sure you use short key combinations that are unlikely to be used normally, but still easy to remember](Outlook-image-uly.png)
```
Comment (tiagov8) [inline rules/autocorrect-in-outlook:27]: Hi @ae-cher-ssw 

This image was not uploaded... so it would be broken if merged

- [ ] 1. Can you upload the image in this PR? OR do you have another PR?

File: rules/autocorrect-in-outlook [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+::: greybox
+
+    #T -> your company’s Terms and conditions. E.g. https://www.ssw.com.au/terms-and-conditions
```
Comment (tiagov8) [inline rules/autocorrect-in-outlook:31]: Hi @ae-cher-ssw 

- [ ] 1. You don't need the indentation here. Please remove
- [ ] 2. We don't use "->" in rules... Can you change to a single dash "-"
- [ ] 3. Make the #T a code: `#T` 
- [ ] 4. Do we need the words "with Me"? I think "Bookings" is enough

File: rules/autocorrect-in-outlook (hunk 1 above)
Comment (ae-cher-ssw) [reply rules/autocorrect-in-outlook]: Done

Comment (ae-cher-ssw) [reply rules/autocorrect-in-outlook]: Done

File: rules/autocorrect-in-outlook (hunk 3 above)
Comment (ae-cher-ssw) [reply rules/autocorrect-in-outlook]: Done

File: rules/autocorrect-in-outlook (hunk 5 above)
Comment (ae-cher-ssw) [reply rules/autocorrect-in-outlook]: Done

**Review by ae-cher-ssw — COMMENTED** [review]
@tiagov8 - making you the reviewer per your request

**Review by tiagov8 — APPROVED** [review]
